<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Onglet</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="eNom"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Lien</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="eLien"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cOnglet">
           <item>
            <property name="text">
             <string>Général</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Utilitaire</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jeux</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cType">
           <item>
            <property name="text">
             <string>Case à cocher</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Liste</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Nom Liste</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="eListe"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="bAjouter">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Liste</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="eText"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="bEnregistrer">
         <property name="text">
          <string>Enregistrer la liste</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
